packages/router/src/lib/operations/status.ts

```diff
@@ -35,7 +21,7 @@ export const getStatus = (_requestContext: RequestContext<string>): StatusRespon
     signerAddress,
     trackerLength,
     activeTransactionsLength,
-    swapPools: chainAssetMap,
+    swapPools: chainAssetSwapPoolMap,
```

Comment:
if this response goes over the wire, this will not be properly formatted because it's a map. needs to be proper json